### ISSUE TITLE
Convert SnapGene DNA files to SBOL

### ISSUE THREAD

**Issue by cjmyers** [SynBioDex/SBOL-Validator#99]
In order to support SnapGene DNA file conversion to SBOL, the uploaded file needs to be saved locally with a ".dna" extension.  Right now, all files get saved with a ".sbol" extension.  The conversion utility in libSBOLj currently is only able to identity SnapGene DNA files from their file extension.

**Comment by zachzundel** [comment]
Is this all that has  to happen? The file just has to be saved with a .dna extension? How can I tell if I've been given a SnapGene file?

**Comment by cjmyers** [comment]
Yes, currently I have no other way to know a file is a dna file than by its file extension.  So, if you receive a file with extension .dna then if you preserve the extension, then it will convert it.
